types/ember/index.d.ts

```diff
@@ -3442,7 +3442,20 @@ declare module '@ember/component/checkbox' {
 declare module '@ember/component/helper' {
     import Ember from 'ember';
     export default class Helper extends Ember.Helper { }
-    export const helper: typeof Ember.Helper.helper;
+    /**
+     * In many cases, the ceremony of a full `Helper` class is not required.
+     * The `helper` method create pure-function helpers without instances. For
```

Comment:
nit `creates`

Reply:
Thanks for reviewing @mfeckie!

These comments were copied directly from the other location in the file, which I assume were copied from [here](https://www.emberjs.com/api/ember/2.18/functions/@ember%2Fcomponent%2Fhelper/helper). I'm leaning towards just letting it match the docs.

As for letting users know that `params[0]` can be undefined, that can hopefully be inferred from the fact that `params` is typed as `any[]`, and `any` includes `undefined`. Again, I took this signature directly from [the source](https://github.com/emberjs/ember.js/blob/v2.18.2/packages/ember-glimmer/lib/helper.ts#L120). I know of no way to type index 0 differently from the other elements of the array. CC @chriskrycho in case he does.